contrib/completion/bash/docker

```diff
@@ -1597,8 +1598,18 @@ _docker_service_rm() {
 _docker_service_ls() {
 	case "$cur" in
 		-*)
+			COMPREPLY=( $( compgen -W "--help --filter --quiet" -- "$cur" ) )
+			;;
+	esac
+}
+
+_docker_service_scale() {
+	case "$cur" in
+		-*)
 			COMPREPLY=( $( compgen -W "--help" -- "$cur" ) )
 			;;
+		*)
+			COMPREPLY=( $(compgen -S "=" -o nospace -W "$(__docker_services $1)" -- "$cur") )
```

Comment:
@albers Do you know why `-o nospace` is not being obeyed?  

This PR will complete `docker service scale m<TAB>` into `docker service scale myservice= `.  But the trailing equals is unacceptable.